protocol.txt

```diff
@@ -123,3 +127,93 @@ MSG_PACKET_V2
                     are used as a ping mechanism between printer and ESP.
                     It keeps the line busy, prevents printer from
                     reseting ESP and conveys Up field.
+
+MSG_SCAN_START
+    This message is only sent by the printer to the ESP.
+
+        0               1               2               3
+        0 1 2 3 4 5 6 7 0 1 2 3 4 5 6 7 0 1 2 3 4 5 6 7 0 1 2 3 4 5 6 7
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+     0 |                                                               |
+       +                            Intron                             +
+     8 |                                                               |
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+    12 |       8       |       0       |               0               |
+       +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+
+    Sends a signal to esp to start a wifi scan. The scan is iterative
+    and result are sent back to the printer periodically. The first
```

Comment:
The `results are sent back periodically` would hint at the ESP sending the list of APs from time to time. Is that really the case? I thought the printer is „asking“ for nth SSID when it needs it.